splink/internals/comparison_library.py

```diff
@@ -535,6 +535,39 @@ def datetime_parse_function(self):
     def cll_class(self):
         return cll.AbsoluteDateDifferenceLevel
 
+class ArrayStringDistance(ComparisonCreator):
+    def __init__(
+            self,
+            col_name: str,
+            distance_threshold_or_thresholds: Union[Iterable[int], int] = [1],
+            distance_function: str = "levenshtein",
```

Comment:
This should probably be a custom type that lists the allowable values, similar to `DateMetricType` above.